src/lib/developer.test.ts

```
@@ -0,0 +1,1710 @@
+        TEST_DEVELOPER_PASSWORD!,
+        TEST_DEVELOPER_NAME!
+      );
+      console.log("Successfully registered test user");
+      cachedLoginResponse = response;
+
+      // Store tokens for subsequent API calls
+      window.localStorage.setItem("access_token", response.access_token);
+      window.localStorage.setItem("refresh_token", response.refresh_token);
+
+      return response;
+    } catch (registerError: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)